engine/api/repositoriesmanager/repogitlab/client.go

```diff
@@ -208,7 +208,7 @@ func (c *GitlabClient) Commit(repo, hash string) (sdk.VCSCommit, error) {
 		DisplayName: gc.AuthorName,
 		Email:       gc.AuthorEmail,
 	}
-	commit.Timestamp = gc.CommittedDate.Unix()
+	commit.Timestamp = gc.AuthoredDate.Unix()
```

Comment:
Could you also change in engine/vcs/gitlab/client_commit.go ?

Reply:
done